Sources/Model/TransferMethod/HyperwalletPaperCheck.swift

```diff
@@ -0,0 +1,469 @@
+        public func addressLine2(_ addressLine2: String) -> Builder {
+            storage[TransferMethodField.addressLine2.rawValue] = AnyCodable(value: addressLine2)
+            return self
+        }
+
+        /// Sets the user's relationship with the bank account holder.
+        ///
+        /// - Parameter relationship: The `RelationshipType`
+        /// - Returns: a self `HyperwalletPaperCheck.Builder` instance.
+        public func bankAccountRelationship(_ relationship: String) -> Builder {
+            storage[TransferMethodField.bankAccountRelationship.rawValue] = AnyCodable(value: relationship)
+            return self
+        }
+
+        /// Sets the bank account holder's city.
+        ///
+        /// - Parameter city: The bank account holder's city.
+        /// - Returns: a self `HyperwalletPaperCheck.Builder` instance.
+        public func city(_ city: String) -> Builder {
+            storage[TransferMethodField.city.rawValue] = AnyCodable(value: city)
+            return self
+        }
+
+        /// Sets the bank account holder's country.
+        ///
+        /// - Parameter country: The bank account holder's country.
+        /// - Returns: a self `HyperwalletPaperCheck.Builder` instance.
+        public func country(_ country: String) -> Builder {
+            storage[TransferMethodField.country.rawValue] = AnyCodable(value: country)
+            return self
+        }
+
+        /// Sets the bank account holder's postal code.
+        ///
+        /// - Parameter postalCode: The bank account holder's postal code.
+        /// - Returns: a self `HyperwalletPaperCheck.Builder` instance.
+        public func postalCode(_ postalCode: String) -> Builder {
+            storage[TransferMethodField.postalCode.rawValue] = AnyCodable(value: postalCode)
+            return self
+        }
+
+        /// Sets the shipping method for paper check.
+        ///
+        /// - Parameter shippingMethod: The bank account holder's postal code.
+        /// - Returns: a self `HyperwalletPaperCheck.Builder` instance.
+        public func shippingMethod(_ shippingMethod: String) -> Builder {
+            storage[TransferMethodField.shippingMethod.rawValue] = AnyCodable(value: shippingMethod)
+            return self
+        }
+
+        /// Sets the bank account holder's state, province or region.
+        ///
+        /// - Parameter stateProvince: The bank account holder's state, province or region.
+        /// - Returns: a self `HyperwalletPaperCheck.Builder` instance.
+        public func stateProvince(_ stateProvince: String) -> Builder {
+            storage[TransferMethodField.stateProvince.rawValue] = AnyCodable(value: stateProvince)
+            return self
+        }
+
+        // Sets the bank account holder's profile type.
+        ///
+        /// - Parameter profileType: The bank account holder's profile type, e.g. INDIVIDUAL or BUSINESS
+        /// - Returns: a self `HyperwalletBankAccount.Builder` instance.
```

Comment:
Update HyperwalletPaperCheck instead of HyperwalletBankAccount everywhere in this class

Reply:
🤦